lib/agenda.js

```diff
@@ -24,6 +24,7 @@ var Agenda = module.exports = function(config, cb) {
   this._lockedJobs = [];
   this._jobQueue = [];
   this._defaultLockLifetime = config.defaultLockLifetime || 10 * 60 * 1000; // 10 minute default lockLifetime
+  this._sort = config.sort || { priority: -1, _id: 1 };
```

Comment:
The indexes created in `db_init` and `handleLegacyCreateIndex` will need updating to account for this. I'd merge the sort config in at the `priority` key's current position.